tasks/elasticsearch-template-insert.yml

```diff
@@ -0,0 +1,36 @@
+#Templates done after restart therefore. e.g. suppose user removes security on a running node and doesn't specify es_api_basic_auth_username and es_api_basic_auth_password.  The templates will subsequently not be removed if we don't wait for the node to restart.
+
+- name: Ensure elasticsearch is started
+  service: name={{instance_init_script | basename}} state=started enabled=yes
+  when: es_start_service
+
+- name: Wait for elasticsearch to startup
+  wait_for: host={{es_api_host}} port={{es_api_port}} delay=10
+  when: es_start_service
+
+- name: Install templates without auth
+  uri:
+    url: "http://{{es_api_host}}:{{es_api_port}}/_template/{{item | filename}}"
```

Comment:
@gingerwizard if we use filename with `.json` extension, should we leave it in the template name (in the url)?

Tips: Since Ansible 2.0 `{{ (item | filename | splitext)[0] }}`